Signal/src/AppDelegate.m

```diff
@@ -248,6 +248,8 @@ - (void)setupEnvironment
 
     self.staleNotificationObserver = [OWSStaleNotificationObserver new];
     [self.staleNotificationObserver startObserving];
+
+    [TSPreKeyManager sharedManager];
```

Comment:
I moved observation of the activation event into `TSPreKeyManager` in order to gather concerns.